Sam

Samlewis99 opened the pull request:
Hey Everyone, here is my updated folder for Seismic Event mapping. If someone could please copy the branch to jupter and review the following:
- map_maker notebook works
- Functions all have documentation
- Comments are readable and where they need to be
- Spelling and grammar
- run_tests confirms functions are working how they should.
- Also, if someone could check out my tests, that would be great- not sure if they are all appropriate or not.

## Thanks for the feedback in advance, look forward to hearing what needs fixing!

g-nym commented on Sam L/src/my_functions.py:
```diff
@@ -10,77 +10,132 @@
     return features
 
+
+## need to add documentation for what basemaps generators can be chosen, and what maps in those can be choosen within those
```
I reckon hashed comments like this can get deleted out. I believe they were intended as instructions for the assignment, but they don't benefit someone running/reading the program. Removing these notes would tidy up  your code.

Samlewis99 replied:
Good point, these were just personal notes I can remove these now

g-nym commented on Sam L/src/functions.py:
```diff
@@ -1,5 +1,10 @@
 # a random placeholder function to be replaced with
```
So, I'm  genuinely curious... it seems this file  isn't doing anything for the actual  program. Should we delete and clean this up? Leave it as a placeholder? It feels like we ought to do something (either delete or comment) to acknowledge that we're not using it.  Thoughts?

Samlewis99 replied:
Completely agree, I will remove this as it serves no purpose for this map maker

g-nym commented on Sam L/src/my_functions.py:
```diff
@@ -0,0 +1,299 @@
+        features.append(cfeature.NaturalEarthFeature('physical', 'lakes', p))
+
+    if ocean:
+        features.append(cfeature.NaturalEarthFeature('physical', 'oean', p))
```
'oean' typo

Samlewis99 replied:
fixed

g-nym commented on Sam L/src/my_functions.py:
```diff
@@ -0,0 +1,299 @@
+    t1 = UTCDateTime(start)
+    t2   = UTCDateTime(end)
+    
+    #cat = client.get_events...
```
can delete this line

Samlewis99 replied:
Have removed this plus another artifact comment for t1 &t2

g-nym commented on Sam L/src/my_functions.py:
```diff
@@ -0,0 +1,299 @@
+        df['Lon [°]'].loc[ii] = cat[ii].origins[0].longitude
+        df['Lat [°]'].loc[ii] = cat[ii].origins[0].latitude
+        df['mag'].loc[ii] = cat[ii].magnitudes[0].mag
+        #df['event_type'].loc[ii] = cat[ii].event_type
```
do you want this line hashed out?

Samlewis99 replied:
Yeah I changed from plotting the event type to plotting a size, easier to work with this too! I will remove the commented out code for clarity

g-nym commented on Sam L/src/my_functions.py:
```diff
@@ -0,0 +1,299 @@
+        #df['event_type'].loc[ii] = cat[ii].event_type
+        df['event_type'].loc[ii] = 3
+    df1 = df.to_numpy()
+    #df1[:,3] = 1
```
hashed?

Samlewis99 replied:
See above

Samlewis99 commented:
Hey Gwen, I have addressed these issues you have commented on! you picked up a lot of artifacts in my code that I was just missing! I will commit the updates to this branch and pull the request, but it should be operational and ok to be merged. We can wait for Xulu to have a quick look first though before we merge into main.
